scheduler/docs/configuration.adoc

```diff
@@ -288,8 +288,8 @@ An example configuration looks like:
 ```clojure
 {...
  :executor {:command "cook-executor"
-            :default-progress-output-file "stdout"
             :default-progress-regex-string "progress: (\d+)(?: )?(.*)"
+            :environment {"COOK_EXECUTOR" "1"}
```

Comment:
What is this for?

Reply:
It was an example. I have replaced it with `{"EXECUTOR_DEFAULT_PROGRESS_OUTPUT_NAME" "stdout"}` as this one we actually use now.